web_requester.go

```diff
@@ -92,14 +92,15 @@ type WebRequesterFactory struct {
 	Hosts                  []string          `yaml:"Hosts"`
 	Headers                map[string]string `yaml:"Headers"`
 	Body                   string            `yaml:"Body"`
+	BodyPath               string            `yaml:"BodyPath"`
 	ExpectedHTTPStatusCode int               `yaml:"ExpectedHTTPStatusCode"`
 	HTTPMethod             string            `yaml:"HTTPMethod"`
 
 	expandedHeaders map[string][]string
 }
 
 // GetRequester returns a new Requester, called for each Benchmark connection.
-func (w *WebRequesterFactory) GetRequester(uint64) bench.Requester {
+func (w *WebRequesterFactory) GetRequester(uint64) (bench.Requester, error) {
```

Comment:
Let's keep `GetRequester` signature as-is, and instead of returning `error` (which you are ignoring BTW :-)) just use `maybePanic`.

Reply:
Done. Sorry for the delay on the PR.